Get_product_data.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import json
+from bs4 import BeautifulSoup
+import os
+
+class Get_product_infor:
```

Comment:
Python class name convention is with `CamelCase`.
Usually, the class name is a noun/pronoun, not a verb (verbs is used for methods)

-> `ProductFetcher`